startup.sh

```diff
@@ -4,13 +4,24 @@ echo $KIMAI
 
 function waitForDB() {
   # Parse sql connection data
-  # todo: port is not used atm
-  DB_TYPE=$(awk -F '[/:@]' '{print $1}' <<< $DATABASE_URL)
-  DB_USER=$(awk -F '[/:@]' '{print $4}' <<< $DATABASE_URL)
-  DB_PASS=$(awk -F '[/:@]' '{print $5}' <<< $DATABASE_URL)
-  DB_HOST=$(awk -F '[/:@]' '{print $6}' <<< $DATABASE_URL)
-  DB_PORT=$(awk -F '[/:@]' '{print $7}' <<< $DATABASE_URL)
-  DB_BASE=$(awk -F '[/?]' '{print $4}' <<< $DATABASE_URL)
+  if [ ! -z "$DATABASE_URL" ]; then
+    DB_TYPE=$(awk -F '[/:@]' '{print $1}' <<< $DATABASE_URL)
+    DB_USER=$(awk -F '[/:@]' '{print $4}' <<< $DATABASE_URL)
+    DB_PASS=$(awk -F '[/:@]' '{print $5}' <<< $DATABASE_URL)
+    DB_HOST=$(awk -F '[/:@]' '{print $6}' <<< $DATABASE_URL)
+    DB_PORT=$(awk -F '[/:@]' '{print $7}' <<< $DATABASE_URL)
+    DB_BASE=$(awk -F '[/?]' '{print $4}' <<< $DATABASE_URL)
+  else
+    DB_TYPE=${DB_TYPE:mysql}
+    if [ "$DB_TYPE" == "mysql" ]; then
+      export DATABASE_URL="${DB_TYPE}://${DB_USER:=kimai}:${DB_PASS:=kimai}@${DB_HOST:=sqldb}:${DB_PORT:=3306}/${DB_BASE:=kimai}"
+    elif [ "$DB_TYPE" == "sqlite" ]; then
+      export DATABASE_URL="${DB_TYPE}://${DB_BASE:=/%kernel.project_dir%/var/data/kimai.sqlite}"
```

Comment:
Is the slash in frontend of `/%kernel.project_dir%/` correct?
Or should it be 
```
      export DATABASE_URL="${DB_TYPE}://${DB_BASE:=%kernel.project_dir%/var/data/kimai.sqlite}"
```
?

Reply:
Yes (I think).  I've tweaked the docs too, but it allows for either relative to absolute path to be passed to the startup.

Reply:
AFAIk `%kernel.project_dir%` is always an absolute path to the kimai directory, so here `/opt/kimai`. If a user wants to move it somewhere else, one could use `DB_BASE=/tmp/kimai.sqlite`. The slash in front of `%kernel.project_dir%` might not be a problem as `/opt/kimai` and `//opt/kimai` will be handled the same, but is unnecessary.

Reply:
I'll snip it anyway, hang on